plugin/sycl/device_manager.h

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright 2017-2022 by Contributors
+ * \file device_manager.h
+ */
+#ifndef XGBOOST_DEVICE_MANAGER_SYCL_H_
+#define XGBOOST_DEVICE_MANAGER_SYCL_H_
+
+#include <vector>
+#include <mutex>
+#include <string>
+
+#include "CL/sycl.hpp"
+#include "xgboost/context.h"
+
+namespace xgboost {
+namespace sycl {
```

Comment:
Are you sure you want to use `sycl` namespace in the first PR? In the future, you will have to keep doing `::sycl` for the global sycl namespace.

Reply:
Yes. This option is not perfect, but looks like the best one.